src/documentation/chunk.rs

```diff
@@ -227,12 +227,14 @@ impl CheckableChunk {
             .iter()
             .skip_while(move |(fragment_range, _)| fragment_range.end <= start)
             .take_while(move |(fragment_range, _)| fragment_range.start <= end)
-            .filter(|(fragment_range, _)| {
+            .filter_map(|(fragment_range, fragment_span)| {
                 // could possibly happen on empty documentation lines with `///`
-                fragment_range.len() > 0
+                if fragment_range.len() > 0 {
```

Comment:
```suggestion
                if fragment_range.is_empty() {
```
is preferable, but I forget about that too all the time..